### ISSUE TITLE
Paypal bug. Transaction with user discounts

### ISSUE THREAD

**Issue by ping86** [Sylius/Sylius#11002]
**Sylius version affected**: 1.6.4

**Description**  
Recently we update to the new version of sylius 1.6.4 and we have a lot of paypal issues with this error:

```
{
   "PAYMENTREQUEST_0_INVNUM":"xxxx-yyyyy",
   "PAYMENTREQUEST_0_CURRENCYCODE":"EUR",
   "PAYMENTREQUEST_0_AMT":19.99,
   "PAYMENTREQUEST_0_ITEMAMT":19.99,
   "L_PAYMENTREQUEST_0_NAME0":"Product 1",
   "L_PAYMENTREQUEST_0_AMT0":19.99,
   "L_PAYMENTREQUEST_0_QTY0":1,
   "L_PAYMENTREQUEST_0_NAME1":"Discount",
   "L_PAYMENTREQUEST_0_AMT1":-10,
   "L_PAYMENTREQUEST_0_QTY1":1,
   "PAYMENTREQUEST_0_PAYMENTACTION":"Sale",
   "AUTHORIZE_TOKEN_USERACTION":"commit",
   "RETURNURL":"https:\/\/www.awesome-site.com\/payment\/capture\/AWESOME_TOKEN_CAPTURE_5544343534543",
   "CANCELURL":"https:\/\/www.awesome-site.com\/payment\/capture\/AWESOME_TOKEN_CAPTURE_5544343534543?cancelled=1",
   "PAYMENTREQUEST_0_NOTIFYURL":"https:\/\/www.awesome-site.com\/payment\/notify\/AWESOME_TOKEN_NOTIFY_5544343534543",
   "TIMESTAMP":"2019-12-13T16:28:33Z",
   "CORRELATIONID":"aaaaaaaaaa",
   "ACK":"Failure",
   "VERSION":"65.1",
   "BUILD":"53842365",
   "L_ERRORCODE0":"10413",
   "L_SHORTMESSAGE0":"Transaction refused because of an invalid argument. See additional error messages for details.",
   "L_LONGMESSAGE0":"The totals of the cart item amounts do not match order amounts.",
   "L_SEVERITYCODE0":"Error"
}
```

We have more than 300 payments with this issue and we can reproduce in our shop.

**Steps to reproduce**  

1) Create a product promotion discount
2) Select paypal as payment method. 

How we can fix this issue?

Thanks.

**Comment by Zales0123** [comment]
Hello @ping86, can you take a look does this PR would fix the issue? https://github.com/Sylius/Sylius/pull/10938

**Comment by ping86** [comment]
@Zales0123 We fix the issue whit PR #10938

Thanks.
